pkg/cmd/roachtest/tests/mixed_version_job_compatibility_in_declarative_schema_changer.go

```diff
@@ -123,6 +123,16 @@ func executeSupportedDDLs(
 		`ALTER TABLE testdb.testsc.t3 VALIDATE CONSTRAINT check_positive_not_valid`,
 	}
 
+	// DDLs supported in V24_2.
```

Comment:
wasn't DSC support for these DDL statements added in 24.1?

Reply:
*facepalm* yes; got the nomenclature mixed up after the previous rename -- will fix